tests/openssl/tests.unsupported

```diff
@@ -0,0 +1,23 @@
+# Trigger abort, always failed.
```

Comment:
Thanks for the comments for why each of them is unsupported, this is always the standard we've wanted for the other 3rdparty tests!

For the abort test, technically, the abort is the pass condition, so we could add it to the suite, but that would require special case logic in the test harness (which I believe some of the other 3rdparty tests may do). Not a priority to add, but I'd clarify the reason is not that it always "fails" but that we don't handling the expected abort.